Dockerfile

```diff
@@ -7,8 +7,7 @@
 
 FROM ubuntu:24.04
 
-# Change this to latest for any PR to this repo
-ARG VERSION 0.1.71
+ARG VERSION
```

Comment:
This won't work, as the value won't be set inside the `deploy` workflow: https://github.com/exercism/yamlscript-test-runner/blob/main/.github/workflows/deploy.yml

If you'd like to store the version in the Makefile, you'd have to read the version from that file within the Dockerfile.

Reply:
Fixed in 072f0e1f5f3cdd7e4a03381882b61e5f12a73456